app/javascript/crayons/Icons/Icon.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+export const Icon = (props) => {
+  const { src: InternalIcon, native, className, ...otherProps } = props;
```

Comment:
```suggestion
export const Icon = ({ src: InternalIcon, native, className, ...otherProps }) => {
```
not a blocking thing, but we could just directly destructure here 🤷‍♀️